Userland/Applications/GamesSettings/CardSettingsWidget.cpp

```diff
@@ -134,13 +134,13 @@ bool CardSettingsWidget::set_card_back_image_path(DeprecatedString const& path)
     return false;
 }
 
-DeprecatedString CardSettingsWidget::card_back_image_path() const
+String CardSettingsWidget::card_back_image_path() const
 {
     auto& card_back_selection = m_card_back_image_view->selection();
     GUI::ModelIndex card_back_image_index = m_last_selected_card_back;
     if (!card_back_selection.is_empty())
         card_back_image_index = card_back_selection.first();
-    return static_cast<GUI::FileSystemModel const*>(m_card_back_image_view->model())->full_path(card_back_image_index);
+    return MUST(String::from_deprecated_string(static_cast<GUI::FileSystemModel const*>(m_card_back_image_view->model())->full_path(card_back_image_index)));
```

Comment:
The preference at the moment is to use `.release_value_but_fixme_should_propagate_errors()` instead of `MUST()` for unhandled OOM errors.